sammba/registration/struct.py

```diff
@@ -121,6 +121,11 @@ def anats_to_common(anat_filenames, write_dir, brain_volume,
         raise ValueError(
             'Registration kind must be one of {0}, you entered {1}'.format(
                 registration_kinds, registration_kind))
+                
+    if registration_kind is 'nonlinear' and len(anat_filenames) < 5:
+        raise ValueError('At least 5 input files are required to make a ' + 
+                         'template by non-linear \n registration. Only ' + 
+                         str(len(anat_filenames)) + ' have been provided.')
```

Comment:
It's better to use `'{0} have been provided.'.format(len(anat_filenames))`

Reply:
OK will change